### DISCUSSION TITLE
what is the difference between floorp waterfox and librewolf

### DISCUSSION THREAD

**Discussion by olumolu** [Floorp-Projects/Floorp#1124] [Q&A]
What is the key difference between them and why floorp is the most advance browser fork project known?

**Comment by yawns** [comment]
Disclaimer: I'm not a Floorp contributer, just a user.

Waterfox and Librewolf are both "privacy focused." I don't know what's different between the two, but as far as I can tell they're mostly standard Firefox with some different defaults (no telemetry, etc.).

Floorp offers a bunch of UI/feature enhancements and a lot of UI customization. I believe everything the other two do can be set in FF (and thus Floorp's) settings, so this isn't to imply Floorp is less secure/private; but in a general sense, its primary focus (to me -- again not speaking for the project) is UI improvement and additional features.
